src/mixins/filing-template-mixin.ts

```diff
@@ -647,7 +648,7 @@ export default class FilingTemplateMixin extends DateMixin {
     if (this.getHasPlanOfArrangement || this.getFileNumber) {
       filing.changeOfRegistration.courtOrder = {
         fileNumber: this.getFileNumber,
-        effectOfOrder: this.getHasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : null,
+        effectOfOrder: this.getHasPlanOfArrangement ? EffectOfOrders.PLAN_OF_ARRANGEMENT : undefined,
```

Comment:
Please verify by filing a change of registration with a court order number and with / without a POA.

Reply:
Tested ok with FM1085277